### PR TITLE
Set up GitHub Actions workflow for Behat

### PR DESCRIPTION
**Pull request by VinceAAU**
Everything should work now. Sorry if some of it seems a little cursed, I have attempted to clear up exactly what is happening through the comments. 

Thank you to Team Emil for writing the setup guide.

### CODE REVIEW

File: .github/workflows/behat.yml [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+          sudo apt install mariadb-server
+          sudo systemctl disable mariadb
+          sudo systemctl stop mariadb
+
+      - name: Exorcises Snap from the system
+        run: |
+          # The snap version of firefox doesn't work with behat, 
+          # so we have to make sure that apt doesn't install it 
+          # via snap.
+          echo ✝️✝️ GO AWAY SNAP GO AWAY GO AWAY ✝️✝️  # This line is necessary
```
Comment (ChristofferGamel) [inline .github/workflows/behat.yml:29]: :(

File: .github/workflows/behat.yml [hunk 2]
```diff
@@ -0,0 +1,282 @@
+name: Behat
+
+on:
+  pull_request:
+    branches: [ "main", "staging"  ]
+  push:
+    branches: [ "vincetheis-behat" ]
```
Comment (ChristofferGamel) [inline .github/workflows/behat.yml:7]: Hvad gør denne her?

Comment (DeeKahy) [reply .github/workflows/behat.yml:7]: Make sure it runs on main and staging, and then it also runs on that behat banch which is our github actions debug branch.

It doesn't so much outside of debugging.

**Comment by DeeKahy** [comment]
@VinceAAU have behat fails. Also please update the branch

**Review by ChristofferGamel — APPROVED** [review]

**Comment by VinceAAU** [comment]
@EmilChri Currently, the CI pipeline can only test PHPUnit and CodeSniffer. This PR adds Behat to the mix, so whenever a Pull Request is made, it must pass the integration tests too.

**Comment by DeeKahy** [comment]
> What exactly is it this pull request does/adds? I dont understand what is happening

@EmilChri turns out vincent did NOT want to give up on behat and ended up solving it. The last problem was Canonical being stupid and automatically downloading the snap firefox package instead of the one we specified.

**Review by DeeKahy — APPROVED** [review]

**Comment by DeeKahy** [comment]
@ZeroConfidence

**Review by EmilChri — APPROVED** [review]

**Review by ZeroConfidence — APPROVED** [review]

**Pull request merged**